app/src/main/java/org/mozilla/fenix/search/SearchController.kt

```diff
@@ -52,6 +54,14 @@ class DefaultSearchController(
 ) : SearchController {
 
     override fun handleUrlCommitted(url: String) {
+        if (url == "about:crashes") {
+            // The list of past crashes can be accessed via "settings > about", but desktop and
```

Comment:
There is also the possibility of pasting "about:crashes" with "fill link from clipboard". Do you think we should intercept that also?

Reply:
Good point!

I don't think it is worth the effort at this point. Even this here is something I didn't really intented to do and was just more for convenience.